sentry/src/test/java/io/sentry/SentryTest.kt

```diff
@@ -11,6 +11,7 @@ import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertTrue
+import org.junit.rules.TemporaryFolder
 class SentryTest {
 
     @BeforeTest
```

Comment:
should we add an `AfterTest` with `Sentry.close()` ?

Reply:
👍